bot/exts/evergreen/space.py

```diff
@@ -125,7 +124,7 @@ async def nasa(self, ctx: Context, *, search_term: Optional[str] = None) -> None
         )
 
     @space.command(name="epic")
-    async def epic(self, ctx: Context, date: Optional[str] = None) -> None:
+    async def epic(self, ctx: Context, date: Optional[str]) -> None:
         """Get one of latest random image of earth from NASA EPIC API. Support date parameter, format is YYYY-MM-DD."""
```

Comment:
```suggestion
        """Get a random image of the earth from the NASA EPIC API. Support date parameter, format is YYYY-MM-DD."""
```

Reply:
Should "Earth" be capitalized because it's a proper noun?

Reply:
Yep, resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/353264e53f54ba32d6f71fc027bf81aca871fdf7.